zio-http/src/main/scala/zhttp/http/URL.scala

```diff
@@ -87,6 +87,35 @@ final case class URL(
 
     copy(kind = location)
   }
+
+  /**
+   * Returns a new java.net.URI representing this URL.
+   */
+  def toJavaURI: Option[java.net.URI] = self.kind match {
```

Comment:
I think this should not return an Option. Relative urls can also be encoded to URI.

Reply:
you're right.
however, java.net.URL does not support relative paths,
should we keep the Option return signature in toJavaURL ?

Reply:
I think we can have both.